en/firefox/whatsnew/whatsnew-fx80.ftl

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/80.0/whatsnew/all/
+
+# html page title
+whatsnew80-page-title = What’s new with { -brand-name-firefox }
+
+whatsnew80-up-to-date-banner = Congrats! You’re using the latest version of { -brand-name-firefox }.
+
+# Main title
+whatsnew80-main-title = Do just about everything, a little faster
+whatsnew80-main-intro = Move from computer to phone without missing a beat. Get { -brand-name-firefox } for mobile.
+whatsnew80-primary-cta-description = Enter your email or phone number for a download link
```

Comment:
This string doesn't match what's on the demo: `Enter your email or phone number`

Reply:
It's still using the default string, I need to push up what we are overriding it with should be the same here in a moment.